v2.1/partitioning.md

```diff
@@ -208,36 +208,20 @@ To set the enterprise license, see [Set the Trial or Enterprise License Key](ent
 
 #### Step 5. Create and apply corresponding zone configurations
 
-Create appropriate zone configurations:
+To create zone configurations and apply them to corresponding partitions, use the [`ALTER PARTITION ... CONFIGURE ZONE`](configure-zone.html) statement:
 
 {% include copy-clipboard.html %}
 ~~~ shell
-$ cat > north_america.zone.yml
-constraints: [+datacenter=us1]
-~~~
-
-{% include copy-clipboard.html %}
-~~~ shell
-$ cat > australia.zone.yml
-constraints: [+datacenter=au1]
-~~~
-
-Apply zone configurations to corresponding partitions:
-
-{% include copy-clipboard.html %}
-~~~ shell
-$ cockroach zone set roachlearn.students_by_list.north_america \
+$ cockroach sql --execute="ALTER PARTITION roachlearn.students_by_list.north_america OF TABLE roachlearn.students_by_list CONFIGURE ZONE USING constraints='[+datacenter=us1]';" \
```

Comment:
This needn't be a CLI command.  Can be SQL since the previous and next sections are also in the SQL shell.